otaclient/app/log_setting.py

```diff
@@ -99,25 +86,15 @@ def configure_logging():
         _logger = logging.getLogger(_module_name)
         _logger.setLevel(_log_level)
 
-    # NOTE(20240306): for only god knows reason, although in proxy_info.yaml,
-    #   the logging_server field is assigned with an URL, and otaclient
-    #   expects an URL, the run.sh from autoware_ecu_system_setup pass in
-    #   HTTP_LOGGING_SERVER with URL schema being removed!?
-    # NOTE: I will do a quick fix here as I don't want to touch autoware_ecu_system_setup
-    #   for now, leave it in the future.
-    if iot_logger_url := os.environ.get("HTTP_LOGGING_SERVER"):
-        # special treatment for not-a-URL passed in by run.sh
-        # note that we only support http, the proxy_info.yaml should be properly setup.
-        if not (iot_logger_url.startswith("http") or iot_logger_url.startswith("HTTP")):
-            iot_logger_url = f"http://{iot_logger_url.strip('/')}"
+    if iot_logger_url := str(proxy_info.logging_server):
         iot_logger_url = f"{iot_logger_url.strip('/')}/"
```

Comment:
Note: after we directly parse the `logging_server` setting from `proxy_info.yaml`, we don't need to read the environmental vars and the special treatment code can be removed.